client/src/App.tsx

```diff
@@ -20,6 +21,7 @@ function App() {
               <Route path="/" element={<Home />} />
               <Route path="/profiles" element={<Profiles />} />
               <Route path="/featured" element={<Featured />} />
+              <Route path="/publicprofile/:id" element={<PublicProfile />} />
```

Comment:
```suggestion
              <Route path="/profiles/:id" element={<PublicProfile />} />
```

I think this routing naming convention makes more sense as a user will be navigating from the profiles page. What do you think?

Reply:
I agree its much better